src/style.css

```diff
@@ -302,6 +302,52 @@ h6 {
   z-index: -1;
 }
 
+/* COLLECTION COMPONENT */
+
+.collection-section {
```

Comment:
jeśli chodzi o nazwy klasy to myślę że możesz kontynuować konwencję z pozostałych komponentów np: .footer__nav-link, .header__nav-list